cassandra/cassandra-prestop.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "About to call the nodetool drain command"
```

Comment:
Is there any need to provide more info like why we are doing it?  I don't know if that has value or not